sdk/ml/azure-ai-ml/azure/ai/ml/_azure_environments.py

```diff
@@ -189,3 +192,130 @@ def _resource_to_scopes(resource):
     """
     scope = resource + "/.default"
     return [scope]
+
+
+_AZUREML_AUTH_CONFIG_DIR_ENV_NAME = 'AZUREML_AUTH_CONFIG_DIR'
```

Comment:
This is some logic we picked up in v1 SDK since it used azure cli vendored code for auth. We should not carry this over to v2 SDK. In v2 SDK we should explicitly ask customer to pass cloud they want to use rather than trying to pick it up from a config file.